src/Server/RestRouterHandlers/TableRestRouterHandler.cpp

```diff
@@ -232,7 +239,9 @@ void TableRestRouterHandler::buildColumnsJSON(Poco::JSON::Object & resp_table, c
 
         if (col_decl.comment)
         {
-            cloumn_mapping_json.set("comment", queryToString(col_decl.comment));
+            String comment = queryToString(col_decl.comment);
+            comment = comment.substr(1, comment.length() - 2);
+            cloumn_mapping_json.set("comment", comment);
```

Comment:
`cloumn_mapping_json.set("comment", comment.substr(1, comment.length() - 2);`

Reply:
Are we sure comment's length is bigger than 2 ?

Reply:
fixed